### ISSUE TITLE
Cargo make treats .DS_Store in workspace directory as another crate

### ISSUE THREAD

**Issue by edmorley** [sagiegurari/cargo-make#591]
### Describe The Bug
When using Cargo workspaces, with the workspace root `Cargo.toml` pointing at a whole directory, if that directory happens to contain files (such as `.DS_Store`) in addition to directories, then running Cargo make against the workspace fails.

### To Reproduce
1. Perform the setup from #584
2. Create a stray file in the workspace packages directory: `touch packages/.DS_Store`
3. Run `cargo make dist`

### Error Stack

```console
$ cargo make dist
[cargo-make] INFO - cargo make 0.35.0
[cargo-make] INFO - Build File: Makefile.toml
[cargo-make] INFO - Task: dist
[cargo-make] INFO - Profile: development
[cargo-make] INFO - Running Task: workspace
/var/folders/sl/fwxwgr4d0pncnv36t27w9dxr0000gq/T/emorley/fsio/pVcTbmFerD.sh: line 3: cd: ./packages/.DS_Store: Not a directory
[cargo-make] ERROR - Error while executing command, exit code: 1
[cargo-make] WARN - Build Failed.
```

Using `--verbose` shows:

```
[cargo-make] VERBOSE - Loaded Cargo.toml: CrateInfo {
    package: None,
    workspace: Some(
        Workspace {
            members: Some(
                [
                    "packages/.DS_Store",
                    "packages/example1",
                    "packages/example10",
                    "packages/example2",
                    "packages/example3",
                    "packages/example4",
                    "packages/example5",
                    "packages/example6",
                    "packages/example7",
                    "packages/example8",
                    "packages/example9",
                ],
            ),
            exclude: None,
        },
    ),
    dependencies: None,
}
...
...
[cargo-make] VERBOSE - Adding Member: .DS_Store Path: packages/.DS_Store
...
[cargo-make] VERBOSE - Workspace Task Script: [
    "cd ./packages/.DS_Store",
    "cargo make --disable-check-for-updates --allow-private --no-on-error --loglevel=verbose --env CARGO_MAKE_CRATE_CURRENT_WORKSPACE_MEMBER=.DS_Store --profile development -- dist",
...
```

**Comment by sagiegurari** [comment]
can you share the cargo toml? because i take the member list from there (with glob support)

**Comment by edmorley** [comment]
It's the one from #584:

> 1. Perform the setup from #584

**Comment by sagiegurari** [comment]
ok got it, so it matches the * glob. 
I'll check it out.

**Comment by sagiegurari** [comment]
@edmorley I think this should now be fixed in 0.35.1 branch. would love your feedback.

**Comment by sagiegurari** [comment]
released. closing issue.
